Source/Core/Constants+Style.swift

```diff
@@ -40,6 +40,10 @@ extension Constants {
                 case fontSize = 15
             }
 
+            enum InputISOptionalLabel: Double {
```

Comment:
Whenever there's a verb, sounds like stuff going on. We could rename to `InputOptionalLabel` or `OptionalInputLabel` and remove that fake expectation.

I also suggested separately to look at content not in terms of _is it optional_, but rather _is it compulsory_. Personally I find that optional has many connotations (including in Swift), so I would avoid the ambiguity by checking if its required / compulsory (maybe more personal preference so feel free to dismiss this last part if disagree).